hockeysdk/src/main/java/net/hockeyapp/android/CrashManager.java

```diff
@@ -793,7 +793,7 @@ private static void saveConfirmedStackTraces(final WeakReference<Context> weakCo
     /**
      * Searches .stacktrace files and returns them as array.
      */
-    private static String[] searchForStackTraces(final WeakReference<Context> weakContext) {
+    protected static String[] searchForStackTraces(final WeakReference<Context> weakContext) {
```

Comment:
I guess we should apply limit here (on sending) as well.
For correct processing existing crashes, I suggest take only 50 oldest crashes and delete others without sending.

Reply:
I suggest doing this as a separate PR to keep the PRs small.